providers/component-protocol/components/list/model.go

```diff
@@ -59,7 +59,7 @@ type (
 		Description      string      `json:"description,omitempty"`
 		BackgroundImgURL string      `json:"backgroundImgURL,omitempty"`
 		KvInfos          []KvInfo    `json:"kvInfos,omitempty"`
-		Selectable       bool        `json:"selectable,omitempty"`
+		Selectable       *bool       `json:"selectable,omitempty"`
```

Comment:
why *bool, please tell me your scenarios.

Reply:
json can not identify false and no value of bool field. for fronted , selectable is required.

Reply:
Yes, I want to know the different between `false` and `no value`.

Reply:
I mean for fronted, true is default value and false is required if item can not be selected, but if `selectable bool` nothing can be transfer .